cmd/provisioning/lookup.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package provisioning
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+
+	"github.com/cisco-open/fsoc/output"
+	"github.com/cisco-open/fsoc/platform/api"
+)
+
+func newCmdLookup() *cobra.Command {
+
+	var lookupCmd = &cobra.Command{
+		Use:              "lookup-tenant",
+		Short:            "Lookup for a tenant by vanity URL",
+		Long:             `Check whether tenant exist and return tenant Id if it does.`,
+		Example:          `  provisioning lookup-tenant --vanityUrl=fsoc-test.saas.appd-test.com`,
+		Args:             cobra.ExactArgs(0),
+		Run:              lookup,
+		TraverseChildren: true,
+	}
+
+	vanityUrlFlag := "vanityUrl"
```

Comment:
nit Is it possible to define a constant or something like "private static final String" field in Java for "vanityUrl"? The value is also used in the next function (lookup).

Reply:
It is not necessary to have a variable on this. The idiomatic use is to just have the constant string right into the String() call below. And if you want to have all string constants somewhere upfront, then you can define it as a `const` at the global scope level and then use it everywhere.
(but see my next comment on possibly not needing a flag in the first place)